test/test_structured_2d.jl

```diff
@@ -32,6 +32,13 @@ isdir(outdir) && rm(outdir, recursive=true)
     end
   end
 
+  @trixi_testset "elixir_advection_coupled_converter.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_advection_coupled_converter.jl"),    
+      l2   = [0.97560175611287, 0.9707973849860191],
+      linf = [1.5703274355039958, 1.6235401582169442],
+      coverage_override = (maxiters=10^5,))
```

Comment:
Do you really need so many steps for coverage?

Reply:
Not really. I have removed coverage_override entirely.